mlflow/models/resources.py

```diff
@@ -96,3 +127,28 @@ def from_resources(
 
         resource_dict["api_version"] = api_version
         return dict(resource_dict)
+
+    @staticmethod
+    def from_dict(data) -> Dict[str, Dict[ResourceType, List[str]]]:
```

Comment:
The inner List is a List of Dicts, right?

```suggestion
    def from_dict(data) -> Dict[str, Dict[ResourceType, List[Dict]]]:
```

Reply:
Ahh yes :) 
My bad.